tests/conftest.py

```diff
@@ -270,3 +271,14 @@ def db_wishlist_item(session):
     if WishListItem.query.get(w.id):
         session.delete(w)
         session.commit()
+
+
+@pytest.fixture(scope="function")
+def edit_profile_form(session, client):
```

Comment:
check out other fixtures in this file, some of them use mimesis instead of actual data.
Check out if this could be applied in your scenario.

Reply:
of course it could be, I am working on it